docs/tutorials/pwm_control_flight_controller.rst

```diff
@@ -97,164 +83,204 @@ can be seen in the Information tab, as shown in the image below. For more inform
+you may want to decrease the timeout when using the modules on a drone. For more information on the timeout feature of Vertiq modules, refer to the :ref:`manual_timeout` section of the Feature Reference Manual.
+
+.. figure:: ../_static/tutorial_images/pwm_flight_controller/control_center_timeout.png
     :align: center
 
-    Tuning Configuration, Red Dots on Important Parameters 
+    Timeout Parameter
+
+Testing the Module
+*******************
+.. warning:: Double check that the module is secured and there is no propeller attached before performing any testing.
 
-The list below provides a description of the important parameters and details on what they should be set to. They are the same in both DSHOT and PWM setups.:
+To test that module is configured to spin and take throtle commands from the flight controller, open the Testing tab in Control Center and look for the *ESC Input* parameter. This parameter commands the module to spin in the same way 
+that a :ref:`hobby protocol <manual_hobby>` used by a flight controller would, i.e. it sends an :ref:`IQUART throttle command <manual_throttle>`., so it is the best way to test if 
+the module is ready to connect with the flight controller. So setting this to 0.5 will send the module a 50% throttle command with the configuration specified in this example. 
+**Make sure the module is secured before attempting to spin it, as it will "jump" if unsecured**. 
 
-* **Max Volts**: This determines the maximum drive voltage of the motor when operating in Voltage mode. All setpoints will be interpreted as a fraction of this maximum voltage. So if Max Volts is 24V, then a 50% 
-  command will drive the motor with 12V. To use Voltage mode, this needs to be set to the desired maximum. For safety reasons, Max Volts is **set to 10V** for this test setup. This keeps the speed of the motor
-  relatively low. When using the motor on an actual drone you will likely want to increase this to match your battery voltage.
-* **Timeout**: This determines the length of the motor's timeout. If it does not hear any messages within that time, it will timeout and beep. If this is below, 1s, it can be difficult to test with the Control Center. 
-  So for the test setup the **Timeout is set to 1.5s**. Depending on how frequently your flight controller sends throttle commands, you may want to decrease this when using the motors on a drone.
+**However, because this test is meant to simulate commands from a flight controller, it is affected by the** :ref:`manual_advanced_arming` **feature. That means that setting this parameter 
+will not cause the module to spin until the module has armed.** By default Vertiq speed modules require 10 consecutive throttle commands between 0% and 7.5% to arm. So to arm your module using the
+*ESC Input* parameter, set the *ESC Input* to 0.05 to send a 5% throttle command as shown below, and click the set arrow 10 times. On the 10th time you click the set arrow, the module should play 
```

Comment:
hmmm, worth putting in a backlog task for sending multiple of a command at once? maybe more of a Control Center 2.0 kind of thing though

Reply:
I threw it in the backlog, probably low on the list of priorities though: https://app.asana.com/0/1157969475718729/1204506323982078/f